tests/routes/cart/getCart.test.js

```diff
@@ -75,7 +75,14 @@ beforeAll(async () => {
     // add item to cart
     const cart = await request(server)
       .post(`/api/store/${storeId}/cart`)
-      .send({ contents: [product1Id, product2Id], agreedPrice: 40, total: 400 })
+      .send({
+        contents: [
+          { product: product1Id, quantity: 3 },
+          { product: product2Id, quantity: 30 }
+        ],
+        agreedPrice: 40,
+        total: 400
+      })
 
```

Comment:
these tests need to be updated, since the currency and quantity fields are required

Reply:
The currency is added dynamically from the backend @aapetsi